### ISSUE TITLE
No pre-commit file?

### ISSUE THREAD

**Issue by iamnewton** [evilmartians/lefthook#158]
Errors out if there is no `.git/hooks/pre-commit`

## Steps to Reproduce

1. Using an existing git project, pull down latest
2. Install lefthook: `npm -i —save-dev @arkweid/lefthook`
3. Copy over existing config

```yaml
commit-msg:
  commands:
    lint-msg:
      run: npx commitlint
                                                                                          
pre-commit:
  commands:
    lint-html:
      glob: "*.{html,tpl}"
      run: npx stylelint {staged_files}
    lint-md:
      glob: "*.{md,markdown}"
      run: npx mdl {staged_files}
    lint-css:
      glob: "*.{css,scss}"
      run: npx stylelint {staged_files}
    lint-js:
      glob: "*.{js,jsx,ts,tsx}"
      run: npx eslint {staged_files}
    test:docs:
      glob: “*.{md, markdown}”
      run: alex
    test-js:
       run: npx jest
```

4. Run following command to install:

```bash
#603: $ ./node_modules/.bin/lefthook install
SYNCING lefthook.yml
SERVED HOOKS: pre-commit, commit-msg, prepare-commit-msg
open /home/newton/code/.git/hooks/pre-commit: no such file or directory
```

**Comment by jimisaacs** [comment]
Does the directory `.git/hooks` exist? I noticed the same an when I `mkdir .git/hooks` it worked.

**Comment by Envek** [comment]
Released in [0.7.4](https://github.com/evilmartians/lefthook/releases/tag/v0.7.4)
